spec/integration/posts_index_spec.rb

```diff
@@ -0,0 +1,77 @@
+require 'rails_helper'
+
+RSpec.describe 'posts#index', type: :feature do
+  describe 'Post' do
+    before(:each) do
+      @user1 = User.create(name: 'Evans', photo: 'Evans.jpeg', bio: 'This is a bio for me Evans', posts_counter: 0)
```

Comment:
* As mentioned by the previous CR, i get errors when i try to run the tests, Kindly fix these errors 
* I suggest you update your readme file with instructions on how to succesfully run the tests 
* Below is a screenshot of the error i get 
![image](https://user-images.githubusercontent.com/52497246/182643946-66a3a772-9061-46d1-bb65-73998dc6b6cf.png)